components/clipboard-button/index.js

```diff
@@ -57,19 +57,26 @@ class ClipboardButton extends Component {
 	}
 
 	getText() {
-		return this.props.text;
+		let text = this.props.text;
+		if ( 'function' === typeof text ) {
+			text = text();
+		}
+
+		return text;
 	}
 
 	render() {
-		const { className, children } = this.props;
+		// Disable reason: Exclude from spread props passed to Button
```

Comment:
I usually use `omit` for this. I can follow this approach for consistency though

Reply:
The ESLint warning is not great, and we've had to disable it elsewhere. I just found this today, and am inclined to enable the option for our configuration:

https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings

`omit` is fine too, but means an extra dependency and also duplicating the references (once in destructure, once in omit keys listing).

Reply:
I guess it wouldn't be a verbatim duplication, since `text` here is in-fact unused. I see some value in leaving `ignoreRestSIblings` as the default `false` behavior, but... it is a convenient omitting technique. Hmm!